v-next/hardhat/src/internal/cli/init/prompt.ts

```diff
@@ -20,6 +20,25 @@ export async function promptForWorkspace(): Promise<string> {
   return workspaceResponse.workspace;
 }
 
+export async function promptForEsm(
+  pathToPackageJson: string,
+): Promise<boolean> {
+  ensureTTY();
+
+  const { default: enquirer } = await import("enquirer");
+
+  const esmResponse = await enquirer.prompt<{ esm: boolean }>([
+    {
+      name: "esm",
+      type: "confirm",
+      message: `Hardhat only supports ESM projects. Would you like to set the type for ${pathToPackageJson} to "module" now?`,
```

Comment:
is this a relative path from the CWD?

Reply:
We have `shortenPath` in `-utils/path`

Reply:
Good call! It's an absolute path. I changed the name to reflect that and called `shortenPath` on it.